### PR TITLE
unit test cases for /pkg/util/misc

### LINKED ISSUE #745: [Test] add unit test for `/arana/pkg/misc/...` package.
**What would you like to be added**:

**Why is this needed**:
add unit test for `/arana/pkg/misc/...` package.

use this tools:
```
go test ./pkg/security/ -v -coverprofile=count.out

go tool cover -html=count.out
```

### PR DESCRIPTION
**Pull request by tanishasaheb15**
**What this PR does**:
This PR is UT for /pkg/util/misc 
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by tanishasaheb15** [comment]
@dongzl ,  can you please approve this workflow approval?

**Comment by dongzl** [comment]
> @dongzl , can you please approve this workflow approval?

OK

File: pkg/util/misc/misc_test.go [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+ */
+
+package misc_test
+
+import (
+	"bytes"
+	"errors"
+	"testing"
+
+	"github.com/arana-db/arana/pkg/util/misc"
```
Comment (dongzl) [inline pkg/util/misc/misc_test.go:25]: Please format the import, use this tool:

```
go get -u github.com/dubbogo/tools/cmd/imports-formatter

imports-formatter
```

**Review by dongzl — CHANGES_REQUESTED** [review]

File: pkg/util/misc/misc_test.go (hunk 1 above)
Comment (tanishasaheb15) [reply pkg/util/misc/misc_test.go:27]: Done

**Comment by tanishasaheb15** [comment]
@dongzl , Thanks for your feedback , i have addressed the comments. can you please review it ?

**Review by baerwang — APPROVED** [review]

**Review by jjeffcaii — APPROVED** [review]

**Comment by tanishasaheb15** [comment]
can you guys please approve the workflow so that i can merge the changes

**Comment by dongzl** [comment]
> can you guys please approve the workflow so that i can merge the changes

Hi @tanishasaheb15 , My WeChat is 13436670847, we can communicate via WeChat

**Review by dongzl — APPROVED** [review]

**Comment by tanishasaheb15** [comment]
@dongzl , @jjeffcaii  looks like i don't have write access to this repo, so not able to merge the code changes. can you guys please help ?

**Comment by dongzl** [comment]
Hi @tanishasaheb15 , we can communicate with `Slack`, the invite link  `https://join.slack.com/t/arana-db/shared_invite/zt-23kx3opq9-s3sFi5r_x0Wfdjno~JbFtA`

**Pull request merged**